tests/e2e/k8s_test.go

```diff
@@ -119,6 +119,17 @@ func applyK8sResourceFromTemplate(kubectlOptions *k8s.KubectlOptions, templateFi
 	applyK8sResourceManifestFromString(kubectlOptions, manifest.String())
 }
 
+// checkExistenceOfK8sResource queries a Resource by it's kind, namespace and name and
+// returns the output of stderr
+func checkExistenceOfK8sResource(
```

Comment:
We already have similar function: https://github.com/banzaicloud/koperator/blob/b44b042f6ade0335d8617fb3b2f7e5230f9cc6c5/tests/e2e/k8s_test.go#LL278C18-L278C18